### DISCUSSION TITLE
Full sized BackgroundImage.

### DISCUSSION THREAD

**Discussion by mattfullerdev** [mantinedev/mantine#2511] [Q&A] [answered]
I'm trying to make a BackgroundImage on the whole screen, full sized and fixed so that when the content is scrolled, the back ground stays in place.

Code started like this:

```
<BackgroundImage src="background.jpg">
  <Box
     sx={{
       height: '100vh',
     }}
     pt={30}
   >
     <Container size={1200}>
       <Paper withBorder shadow="md" p={30} radius="md">
```

I've tried background size and background attachment but it has no effect. What am I missing?

Thanks

**Comment by rtivital** [comment, marked as answer]
`<BackgoundImage sx={{ position: 'fixed', inset: 0 }} />`

**Reply by mattfullerdev** [reply]
Thanks
